uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/AadLoginComponentHandler.java

```diff
@@ -157,4 +157,9 @@ public void handleRegistration() {
 
         handleNextButton();
     }
+
+    public void handleSignInWithDifferentAccount(String username, String password) {
```

Comment:
Can probably put this on `IMicrosoftStsLoginComponentHandler`

Reply:
or perhaps ADFS doesn't support it? in that case we can leave it here

Reply:
I wasn't sure whether it did.  I can move it.